File: Project Academy/02. ViewModel/java/app/src/main/java/com/dicoding/academies/ui/bookmark/BookmarkViewModel.java
package com.dicoding.academies.ui.bookmark;

import androidx.lifecycle.ViewModel;

import com.dicoding.academies.data.CourseEntity;
import com.dicoding.academies.utils.DataDummy;

import java.util.List;

public class BookmarkViewModel extends ViewModel {
    List<CourseEntity> getBookmarks() {
        return DataDummy.generateDummyCourses();
    }
}

File: Project Academy/02. ViewModel/kotlin/app/src/main/java/com/dicoding/academies/ui/bookmark/BookmarkViewModel.kt
package com.dicoding.academies.ui.bookmark

import androidx.lifecycle.ViewModel

import com.dicoding.academies.data.CourseEntity
import com.dicoding.academies.utils.DataDummy

class BookmarkViewModel : ViewModel() {

    fun getBookmarks(): List<CourseEntity> = DataDummy.generateDummyCourses()
}

